pkg/exporter/cassandra/config.go

```diff
@@ -0,0 +1,39 @@
+package cassandra
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config hold configuration of Jaeger Elasticserch exporter/storage.
+// TODO consider using cassandra.Options directly from Jaeger
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	Servers                       []string                 `mapstructure:"servers"`
+	Port                          int                      `mapstructure:"port""`
+	Keyspace                      string                   `mapstructure:"keyspace"`
+	LocalDC                       string                   `mapstructure:"local_dc"`
+	Consistency                   string                   `mapstructure:"consistency"`
+	ProtocolVersion               int                      `mapstructure:"proto_version"`
+	ConnectionsPerHost            int                      `mapstructure:"connections_per_host"`
+	ConnectTimeout                time.Duration            `mapstructure:"connection_timeout"`
+	ReconnectInterval             time.Duration            `mapstructure:"reconnect_interval"`
+	SocketKeepAlive               time.Duration            `mapstructure:"socket_keep_alive"`
+	SpanStoreWriteCacheTTL        time.Duration            `mapstructure:"span_store_write_cache_ttl"`
+	MaxRetryAttempts              int                      `mapstructure:"max_retry_attempts"`
+	DisableCompression            bool                     `mapstructure:"disable_compression"`
+	Username                      string                   `mapstructure:"username"`
+	Password                      string                   `mapstructure:"password"`
+	Index                         IndexConfig              `mapstructure:"index"`
+}
```

Comment:
> Also, what is happening about the cassandra.tls.* flags?

This will be done in all storages at once as part of https://github.com/jaegertracing/jaeger-opentelemetry-collector/issues/33

> cassandra.timeout doesn't seem to be included.

This is reader configuration

> enable-dependencies-v2 

This is reader configuration

Archive storage is used by Jaeger query only.